images/runtime/php/__versions.sh

```diff
@@ -3,4 +3,5 @@
 declare -r __DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 source "$__DIR/../../../build/__phpVersions.sh"
 declare -r VERSION_ARRAY=($PHP73_VERSION $PHP72_VERSION $PHP70_VERSION $PHP56_VERSION)
-declare -r VERSION_ARRAY_BUSTER=($PHP74_VERSION $PHP80_VERSION $PHP81_VERSION)
\ No newline at end of file
+declare -r VERSION_ARRAY_BUSTER=($PHP80_VERSION $PHP81_VERSION)
+declare -r VERSION_ARRAY_BULLSEYE=($PHP74_VERSION)
```

Comment:
Can we have tests for PHP 7.4 bullseye based runtime image? I'm not sure if we need to modify anything inside that image other than renaming the dockerfile. It's good to add tests to make sure everything works fine.